src/binding/petsc4py/src/petsc4py/PETSc/DMUtils.pyx

```diff
@@ -9,53 +9,198 @@ cdef class DMInterpolation:
     def __dealloc__(self):
         self.destroy()
 
-    def create(self, comm=None):
+    def create(self, comm: Comm | None = None) -> None:
```

Comment:
This method does not follow conventions elsewhere. Not your fault, Marcin. I'll add two suggestions to fix it.

```suggestion
    def create(self, comm: Comm | None = None) -> Self:
```